package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "coffechat",
```

Comment:
At this point, I'm unsure if `coffechat` is just a typo that has popped up repeatedly, or it's actually the name of the project 😜

Reply:
Oh, man. It's a quirk of my Spanglish brain. Cafe/coffee merge in my mind, so I keep repeating the same mistake. I def, should fix.